documentation/staging/content/userguide/istio/istio.md

```diff
@@ -110,6 +110,11 @@ $ kubectl --namespace weblogic-operator get pod weblogic-operator-xxx-xxx -o yam
 
 In the second command, change `weblogic-operator-xxx-xxx` to the name of your pod.
 
+When using OpenShift service mesh, because it does not support namespace-wide Istio sidecar injection, you must set the
```

Comment:
Can we add a comment earlier before line 94 (label namespace),  skip the labeling for OpenShift service mesh and move 113-117 after the paragraph 98-100 ?   Basically, for non openshift the steps are  label namespace and regular install of the operator,  but for openshift service mesh, just install but with the set annotations command.   I think this will be more clear.